releasenotes/notes/53318.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: release-notes/v2
+kind: feature
+area: installation
+issue:
+  - 43204
+releaseNotes:
```

Comment:
nit: should have an 'upgrade note' so its on the short list of breaking changes. Probably containing most of this and make the release note ~1 line